components/03-modules/side-menu/SideMenu.js

```diff
@@ -0,0 +1,74 @@
+import get from 'lodash.get'
+import AlpacaIcon from '../../01-globals/icon/Icon.vue'
+
+export default {
+  components: {
+    AlpacaIcon
+  },
+  props: {
+    menuItems: {
+      type: Array,
+      required: true
+    },
+    rootPositionSign: {
+      type: String,
+      default: '*'
```

Comment:
why `*`?

Reply:
It was a way to distinguish root level from child levels, as mentioned below I found a simpler way to do it.